### PR TITLE
Support generate plugin list index config in menu file

### PR DESCRIPTION
**Pull request by mrproliu**
* Support generate plugin list index to `menu.yml`
* Fix wrong contribute plugin docs.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,5 +15,6 @@ require (
 	go.uber.org/automaxprocs v1.4.0
 	google.golang.org/grpc v1.38.0
 	google.golang.org/protobuf v1.26.0
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
```
Comment (kezhenxu94) [inline go.mod:18]: Usually , a new item in `go.mod` requires a new item in `LICENSE` file

Comment (mrproliu) [reply go.mod:18]: Thx, have added.

Comment (wu-sheng) [reply go.mod:18]: Is this for satellite runtime? If not, we don't need the license.

File: LICENSE [hunk 2]
```diff
@@ -215,6 +215,7 @@ Apache 2.0 licenses
 The following components are provided under the Apache License. See project link for details.
 The text of each license is the standard Apache 2.0 license.
 
+go-yaml (yaml) v3.0.0: https://github.com/go-yaml/yaml Apache 2.0
```
Comment (kezhenxu94) [inline LICENSE:218]: This is binary dependency, should be in `dist/LICENSE` file

File: go.mod (hunk 1 above)
Comment (kezhenxu94) [reply go.mod:18]: > Is this for satellite runtime? If not, we don't need the license.

It's built into the binary, although we only use it to generate docs

Comment (kezhenxu94) [reply go.mod:18]: > > Is this for satellite runtime? If not, we don't need the license.
> 
> It's built into the binary, although we only use it to generate docs

Also, the other PR #42  will use this at runtime, I think it's ok to have this license for now, no matter which PR gets merged first.

Comment (mrproliu) [reply go.mod:18]: Should I need to use the YAML framework v2 version? I have checked, the result basically the same.

Comment (kezhenxu94) [reply go.mod:18]: > Should I need to use the YAML framework v2 version? I have checked, the result basically the same.

Let's use v3, I think newer version is preferable

**Review by kezhenxu94 — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]
Let's try whether this works for website tool.

**Pull request merged**